IntegrationTestSuite/WebservicesIntegrationTests/Tests/EngineeringModel/Option/OptionTestFixture.cs

```diff
@@ -18,42 +18,63 @@
     using System.Linq;
+
     using Newtonsoft.Json;
-    using NUnit.Framework;
     using Newtonsoft.Json.Linq;
-    
+
+    using NUnit.Framework;
+
     [TestFixture]
     public class OptionTestFixture : WebClientTestFixtureBaseWithDatabaseRestore
     {
         [Test]
         public void VerifyThatNewOptionCanBeCreatedWithWebApi()
         {
-            var iterationUri = new Uri(string.Format(UriFormat, this.Settings.Hostname,"/EngineeringModel/9ec982e4-ef72-4953-aa85-b158a95d8d56/iteration/e163c5ad-f32b-4387-b805-f4b34600bc2c"));
+            var iterationUri = new Uri(string.Format(UriFormat, this.Settings.Hostname, "/EngineeringModel/9ec982e4-ef72-4953-aa85-b158a95d8d56/iteration/e163c5ad-f32b-4387-b805-f4b34600bc2c"));
             var postBodyPath = this.GetPath("Tests/EngineeringModel/Option/PostNewOption.json");
 
-            var postBody = base.GetJsonFromFile(postBodyPath);
+            var postBody = this.GetJsonFromFile(postBodyPath);
             var jArray = this.WebClient.PostDto(iterationUri, postBody);
 
-            var engineeeringModel = jArray.Single(x => (string)x[PropertyNames.Iid] == "9ec982e4-ef72-4953-aa85-b158a95d8d56");
-            Assert.AreEqual(2, (int)engineeeringModel[PropertyNames.RevisionNumber]);
+            var engineeeringModel = jArray.Single(x => (string) x[PropertyNames.Iid] == "9ec982e4-ef72-4953-aa85-b158a95d8d56");
+            Assert.AreEqual(2, (int) engineeeringModel[PropertyNames.RevisionNumber]);
 
-            var iteration = jArray.Single(x => (string)x[PropertyNames.Iid] == "e163c5ad-f32b-4387-b805-f4b34600bc2c");
-            Assert.AreEqual(2, (int)iteration[PropertyNames.RevisionNumber]);
+            var iteration = jArray.Single(x => (string) x[PropertyNames.Iid] == "e163c5ad-f32b-4387-b805-f4b34600bc2c");
+            Assert.AreEqual(2, (int) iteration[PropertyNames.RevisionNumber]);
             var expectedOptions = new List<OrderedItem> { new OrderedItem(1, "bebcc9f4-ff20-4569-bbf6-d1acf27a8107"), new OrderedItem(2, "e90e4bcd-6e17-4b75-80fb-6cded78bed57") };
             var optionsArray = JsonConvert.DeserializeObject<List<OrderedItem>>(iteration[PropertyNames.Option].ToString());
             CollectionAssert.AreEquivalent(expectedOptions, optionsArray);
 
-            var option = jArray.Single(x => (string)x[PropertyNames.Iid] == "e90e4bcd-6e17-4b75-80fb-6cded78bed57");
-            Assert.AreEqual(2, (int)option[PropertyNames.RevisionNumber]);
+            var option = jArray.Single(x => (string) x[PropertyNames.Iid] == "e90e4bcd-6e17-4b75-80fb-6cded78bed57");
+            Assert.AreEqual(2, (int) option[PropertyNames.RevisionNumber]);
             Assert.AreEqual("test option", (string) option[PropertyNames.Name]);
-            Assert.AreEqual("testoption", (string)option[PropertyNames.ShortName]);
+            Assert.AreEqual("testoption", (string) option[PropertyNames.ShortName]);
+        }
+
+        [Ignore("Reordering options is not possible in Webservice")]
```

Comment:
i propose we add a GH ticket reference to fix this

Reply:
Was already done. Added the issue URL to the text in the Ignore Attribute.